src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -449,19 +444,16 @@
     <value>Uniform to fill</value>
   </data>
   <data name="Profile_CloseOnExit.Header" xml:space="preserve">
-    <value>How the profile closes</value>
-  </data>
-  <data name="Profile_CloseOnExit.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Sets how the profile reacts to termination or failure to launch. Graceful will close when exit is typed or the process exits normally.</value>
+    <value>Profile termination behavior</value>
   </data>
   <data name="Profile_CloseOnExitAlways.Content" xml:space="preserve">
-    <value>Always</value>
+    <value>Always close</value>
   </data>
   <data name="Profile_CloseOnExitGraceful.Content" xml:space="preserve">
-    <value>Graceful</value>
+    <value>Close when exit is typed/process exits normally</value>
   </data>
   <data name="Profile_CloseOnExitNever.Content" xml:space="preserve">
-    <value>Never</value>
+    <value>Never close</value>
   </data>

```

Comment:
Tagging @DHowett since he's the creator of this setting. Curious of his thoughts.

Reply:
Here's a thought.

* Close when process exits, fails or crashes
* Close when process exits successfully
* Never automatically close

we could add a subtitle that expresses "This setting controls when terminal panes will automatically close." if you want.

Reply:
edited to add "only"